src/main/java/com/openlattice/shuttle/util/Constants.java

```diff
@@ -52,4 +52,5 @@
     public static final String LENGTH              = "length";
     public static final String PRE                 = "pre";
     public static final String CUTOFF              = "cutoff";
+    public static final Integer DECADE_CUTTOFF     = 20;
```

Comment:
would prefer using the unboxed type (`int` vs `Integer`)

Reply:
ok done, also fixed that typo 😛